src/App.js

```diff
@@ -10,18 +10,23 @@ import Footer from './Components/Footer'
 import User from './Components/UserProfile/User'
 import Home from './Components/Home/Home'
 
+import user from './config/user.json'
+
 function App() {
   return (
     <Router>
       <div className="m-2 md:m-4">
-        <Switch>
-          <Route path="/:username">
-            <User />
-          </Route>
-          <Route path="/">
-            <Home />
-          </Route>
-        </Switch>
+        {user.username && <User singleUser={user} />}
```

Comment:
Had a look at this from the desktop, I have a question, if someone visits `${site}/a_page_that_doesnot_exist`, will they not receive a 404 or redirection?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
Currently no, but they shouldn't do it anyway 🙈

Reply:
Good spot, as there is only 1 url, tbh I think catching all here and not showing a 404 is probably a good thing IMHO